index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <!-- header -->
+    
+    <div class="container">  <!-- wrapper -->
+        
+        <header>
+            <nav class="navbar">
+                <a class="logo" href="#"><img src="./images/Logo.png" alt="Image-logo missing"></a>
+                <a class="bar" href="#"><img src="./images/Union.png" alt="Image-bar  missing"></a>
+            </nav>
+        </header>
+
+        <!-- small-container -->
+        <section class="small-container">
+            <h1>
+                I’m Michael <br>
+                Glad to see you!
+            </h1>
+            <p class="paragraph-text">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+
+            <a href="#" class="social-link">
+                Let's connect
+            </a>
+
+            <!-- social-media-link -->
+            
+            <ul class="social-media-link">
+                <li class="social-icon">
+                    <a href="#">
+                        <img src="./images/Vector-1.png" alt="Image missing">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#">
+                        <img src="./images/Linkedin icon.png" alt="Image missing">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#">
+                        <img src="./images/Vector.png" alt="Image missing">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#">
+                        <img src="./images/Vector.svg" alt="Image missing">
+                    </a>
+                </li>
+            </ul>
+        </section>
```

Comment:
- [ ] Kindly remember, it's a good practice to always close ```tags``` properly in HTML to help ensure that the code is **clean, readable, maintainable, and accessible** for everyone.
    - Kindly add a closing tag for ```</body``` and also for ```</html>``` 😃